corehq/apps/users/management/commands/fix_commcare_version.py

```diff
@@ -0,0 +1,57 @@
+from django.core.management.base import BaseCommand
+from corehq.apps.users.dbaccessors import get_all_user_rows, get_mobile_user_count
+from corehq.apps.users.models import CouchUser, CommCareUser
+from corehq.util.log import with_progress_bar
+from corehq.apps.es import DomainES, filters
+
+
+class Command(BaseCommand):
+    help = 'Fix truncated commcare_version for mobile users'
+
+    def handle(self, *args, **kwargs):
+        self.update_commcare_version()
+
+    def update_commcare_version(self):
+        total_update = 0
+        total_users = 0
+        active_domains = DomainES().is_active().values_list('name', flat=True)
+
+        for domain in with_progress_bar(active_domains, len(active_domains)):
+            total_users_in_domain = get_mobile_user_count(domain)
+            total_users += total_users_in_domain
+            users = get_all_user_rows(domain, include_web_users=False, include_mobile_users=True,
+                                      include_inactive=True, include_docs=True)
+
+            for user in users:
+                user = CouchUser.wrap_correctly(user['doc'])
+
+                changes_made = False
+                if getattr(user, 'devices', None):
+                    changes_made |= self.update_version(user.last_device, 'commcare_version')
```

Comment:
Why do we need bitwise OR here? I think if `self.update_version` returns true we will save the user, right?

Reply:
Right, for the first call of `update_version`, bitwise OR isn't necessary...

Reply:
Is it necessary for other calls?